rudderanalytics-client/src/main/java/com/rudderstack/sdk/java/RudderAnalytics.java

```diff
@@ -44,6 +44,7 @@
     private final List<MessageTransformer> messageTransformers;
     private final List<MessageInterceptor> messageInterceptors;
     private final Log log;
+    final static FlushBlock flushBlock = FlushBlock.create();
```

Comment:
Can we create this instance on request? I mean, if someone wants to turn on synchronization using the Builder flag, then initiate `flushBlock`.

Reply:
Implemented